src/js/components/terms-inspector.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * WordPress dependencies
+ */
+import { select, subscribe } from '@wordpress/data';
+
+const Terms = {
+	wrapper: null,
+	query: {
+		per_page: -1,
+		orderby: 'name',
+		order: 'asc',
+		_fields: 'id,name,parent',
+		context: 'view',
+	},
+	available: {},
+	_init() {
+		this.wrapper = document.getElementById( 'cld-tax-items' );
+		setTimeout( () => {
```

Comment:
The getTaxonomies() never returns null, since it's part of the core data.

The 3sec wait is for the lookup of the `getEntityRecords` to ensure that the core request fires first, and to prevent to many events being triggered.

That's why I have the isResolving within it.